test/clojider_gatling_highcharts_reporter/reporter_test.clj

```diff
@@ -21,17 +21,17 @@
                {:id 1 :name "Request2" :start 1391936497299 :end 1391936497996 :result false}]}])
 
 (def expected-lines
-  ["clj-gatling\tmySimulation\tRUN\t1391936496000\t\u0020\t2.0"
-   "Test scenario\t1\tUSER\tSTART\t1391936496814\t1"
-   "Test scenario\t1\tREQUEST\t\tRequest1\t1391936496853\t1391936496853\t1391936497299\t1391936497299\tOK\t\u0020"
-   "Test scenario\t1\tREQUEST\t\tRequest2\t1391936497299\t1391936497299\t1391936497996\t1391936497996\tKO\t\u0020"
-   "Test scenario\t1\tUSER\tEND\t1391936496814\t1391936496814"])
+  ["RUN\tclj-gatling\tmySimulation\t1391936496000\t\u0020\t3.7.4"
```

Comment:
Unfortunately this test is dependent on users time zone.  The problem is this line here: https://github.com/mhjort/clojider-gatling-highcharts-reporter/blob/master/src/clojider_gatling_highcharts_reporter/reporter.clj#L33 where `ZoneId/systemDefault` is used. 

Suggestion. Move date handling functions to own namespace (e.g. `date-util`). I think all we need is `local-now` and `system-default-zone-id`. Then you can use redefs for those in tests (tests will pass if default zone id is UTC).

Reply:
`(ZonedDateTime/now)` produces the same result as `(-> (LocalDateTime/now) (.ofZone (ZoneId/systemDefault)))`, so I've changed it so that `start-time` in `core` is a `ZonedDateTime` directly. Given we don't do anything with the start time except immediately pass it into the CSV function where it is converted from Local -> Zoned via `.ofZone`, this makes no functional difference and allows us to redef `start-time` for the test so that it produces a value that's always in UTC regardless of the machine's local timezone.